src/App.js

```diff
@@ -11,12 +11,60 @@ import "./App.css";
 
 function App() {
   const [currentUser, setCurrentUser] = useState(null);
+  const [currentAvatar, setCurrentAvatar] = useState(null);
+
+  const itemData = [
+    {
+      id: 1,
+      img: "dino-idle.png",
+      title: "original dino",
+    },
+    {
+      id: 2,
+      img: "dino-baseball.png",
+      title: "baseball dino",
+    },
+    {
+      id: 3,
+      img: "dino-disco.png",
+      title: "disco dino",
+    },
+    {
+      id: 4,
+      img: "dino-mariachi.png",
+      title: "mariachi dino",
+    },
+    {
+      id: 5,
+      img: "dino-rainbow.png",
+      title: "rainbow dino",
+    },
+    {
+      id: 6,
+      img: "dino-sigma.png",
+      title: "sigma dino",
+    },
+    {
+      id: 7,
+      img: "dino-spiderman.png",
+      title: "spiderman dino",
+    },
+  ];
+
+  const baseUrl = "https://chrome-dino-game.s3.amazonaws.com/assets/";
+
   const changeUser = (username) => {
     setCurrentUser(username);
   };
 
-  const profile =
-    "https://chrome-dino-game.s3.amazonaws.com/assets/dino-idle.png";
+  const changeProfileAvatar = (avatar) => {
+    console.log(`${baseUrl}${itemData[avatar - 1]["img"]}`);
+    setCurrentAvatar(`${baseUrl}${itemData[avatar - 1]["img"]}`);
```

Comment:
I think using 'avatar - 1' renders the id useless - might be worth removing the id keys from each object in itemData